app/models/casa_case.rb

```diff
@@ -12,6 +12,8 @@ class CasaCase < ApplicationRecord
 
   has_many :case_assignments, dependent: :destroy
   has_many(:volunteers, through: :case_assignments, source: :volunteer, class_name: "User")
+  has_many :active_case_assignments, -> { is_active }, class_name: "CaseAssignment"
+  has_many :assigned_volunteers, -> { active }, through: :active_case_assignments, source: :volunteer, class_name: "Volunteer"
```

Comment:
I wish these two column names matched but I don't think it's worth the migration nuisance unless someone wants to do it for fun. So it goes.